### ISSUE TITLE
Should use text/template for release notes

### ISSUE THREAD

**Issue by bep** [goreleaser/goreleaser#276]
Hugo uses examples in Markdown in the release notes, one example:

```markdown
---
title: "{{ replace .TranslationBaseName "-" " " | title }}"
date: {{ .Date }}
draft: true
---

**Insert Lead paragraph here.**

<!--more-->

## New Cool Posts

{{ range first 10 ( where .Site.RegularPages "Type" "cool" ) }}
* {{ .Title }}
{{ end }}
```

Since goreleaser uses `html/template` to handle these release notes, it escapes the brackets and quotes in the above example, and I have to manually edit the release notes to make it look right.

Not sure if there is a specific reason why not to just use `text/template` here, which, for us, would be a better choice.

https://github.com/gohugoio/hugo/issues/3622

**Comment by caarlos0** [comment]
no particular reason, it's a bug.

**Comment by caarlos0** [comment]
fixed on v0.22.2 :)
